src/__tests__/client.test.js

```diff
@@ -7,7 +7,9 @@ let accountClient = null
 let registrationToken = null
 let registeredClients = []
 
-jest.setTimeout(30000)
+// default of 5s is sometimes, but not always, enough time. give the tests 10s
+let testTimeout = Number(process.env.TEST_TIMEOUT)
+jest.setTimeout(testTimeout)
```

Comment:
✂️ these from the test files. the global test timeout is configurable in `jest.config.test` and should be extended there 😀

Reply:
`import { testTimeout } from '../../jest.config'

// default of 5s is sometimes, but not always, enough time. give the tests 10s
jest.setTimeout(testTimeout)`

This does no work, it was my original attempt. But for some reason it does not find it,

Reply:
ah, sorry for the misdirection. jest loads up the configuration when it runs (through `npm test`). all that's needed is to increase the value in the actual file and everything will just magically work. no need to import or refer to timeouts in the test files themselves.